actors/builtin/multisig/multisig_actor.go

```diff
@@ -328,6 +379,62 @@ func (a Actor) approveTransaction(rt vmr.Runtime, txnID TxnID) {
 	}
 }
 
+func (a Actor) approveTransactionWithHash(rt vmr.Runtime, params *TxnIDParams) {
```

Comment:
I think we can just delete the old `approveTransaction` method, right?

Reply:
We can't as written. `approveTransaction` is used to institute the `PROPOSE` approval. There won't be a hash to check. That said, I can rewrite the code such that it uses a boolean to bypass the hash check in such a case. I'll do that.